### DISCUSSION TITLE
Git with ssh for wiki pages?

### DISCUSSION THREAD

**Discussion by johanberntsson** [community/community#23901] [New to GitHub]
<p>Hi, I have a registered ssh key on my account , and can do git over ssh without trouble for my normal repositories. But when I go to my project’s wiki page I only see a https git url, and no ssh option. This means that I cannot manage my wiki pages using git and ssh. Is it possible to do somehow? I prefer to use the same method (ssh) everywhere.</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hi <a class="mention" href="/johanberntsson">@johanberntsson</a> and welcome! Have you tried cloning <code>git@github.com:&lt;username&gt;/&lt;repo&gt;.wiki.git</code> over SSH?</p>

**Comment by jsoref** [comment]
<aside class="onebox githubissue" data-onebox-src="https://github.com/github/docs/issues/9417">
  <header class="source">

      <a href="https://github.com/github/docs/issues/9417" target="_blank" rel="noopener">github.com/github/docs</a>
  </header>

  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Issue">
	  <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/github/docs/issues/9417" target="_blank" rel="noopener">explain how to clone github wikis via ssh</a>
    </h4>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2021-08-29" data-time="19:55:51" data-timezone="UTC">07:55PM - 29 Aug 21 UTC</span>
      </div>


      <div class="user">
        <a href="https://github.com/jsoref" target="_blank" rel="noopener">
          <img alt="jsoref" src="https://user-images.githubusercontent.com/2119212/181079908-27d9254c-2410-4434-8c32-c886a23322d6.png" class="onebox-avatar-inline" width="20" height="20">
          jsoref
        </a>
      </div>
    </div>

    <div class="labels">
        <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">
          content
        </span>
    </div>
  </div>
</div>

  <div class="github-row">
    <p class="github-body-container">### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [C<span class="show-more-container"><a href="" rel="noopener" class="show-more">…</a></span><span class="excerpt hidden">ode of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://github.com/github/docs/blob/384d146375f49f0cd0dc0ab144dfd4255d99f819/content/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages.md

### What part(s) of the article would you like to see updated?

Cloning wikis to your computer

could also explain how to use ssh to clone wikis

### Additional information

https://github.community/t/git-with-ssh-for-wiki-pages/198311</span></p>
  </div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by johanberntsson** [comment]
<p>Thanks, it worked. I thought I tried it, but I must have had some typing mistake somewhere. My problem is now solved.</p>
